TypeScript error after add `v-wave/nuxt` module

Hello, if we add `v-wave` in the nuxt modules and check the types with `vue-tsc --noEmit`,  we have 2 errors : 

```
node_modules/v-wave/nuxt/index.ts:2:42 - error TS2307: Cannot find module '../src/options' or its corresponding type declarations.

2 import type { IVWavePluginOptions } from '../src/options'
                                           ~~~~~~~~~~~~~~~~

node_modules/v-wave/nuxt/runtime/plugin.ts:7:66 - error TS2559: Type '{ directive: string; }' has no properties in common with type 'Partial<IVWaveDirectiveOptions>'.

7   const { vWave, vWaveTrigger } = VWave.createLocalWaveDirective(options, app.vueApp)
                                                                   ~~~~~~~
```

Reproduction : https://github.com/erwanjugand/v-wave-test (Nuxt 3.13.1, Typescript  5.6.2,  vue-tsc 2.1.6)


There are no problems in v2 :)

--- a/nuxt/index.ts
+++ b/nuxt/index.ts
@@ -1,5 +1,5 @@
-import { defineNuxtModule, addPlugin, createResolver } from '@nuxt/kit'
-import type { IVWavePluginOptions } from '../src/options'
+import { addPlugin, createResolver, defineNuxtModule } from '@nuxt/kit'
+import { DEFAULT_PLUGIN_OPTIONS, type IVWavePluginOptions } from '../src/options'
 
 type ModuleOptions = Partial<IVWavePluginOptions>
 
@@ -9,17 +9,17 @@ export default defineNuxtModule<ModuleOptions>({
     configKey: 'vWave',
     compatibility: {
       nuxt: '>=3.0.0',
-      bridge: true
-    }
+      bridge: true,
+    },
   },
   defaults: {
-    directive: "wave"
+    ...DEFAULT_PLUGIN_OPTIONS,
   },
   setup(options, nuxt) {
     const { resolve } = createResolver(import.meta.url)
 
-    nuxt.options.runtimeConfig.public.vWave = options
+    nuxt.options.runtimeConfig.public.vWave = options as unknown as typeof nuxt.options.runtimeConfig.public.vWave
 
     addPlugin(resolve('./runtime/plugin.ts'))
-  }
+  },
 })

--- a/nuxt/runtime/plugin.ts
+++ b/nuxt/runtime/plugin.ts
@@ -10,13 +10,13 @@ export default defineNuxtPlugin((app) => {
     ...vWave,
     getSSRProps() {
       return {}
-    }
+    },
   })
 
   app.vueApp.directive(`${options.directive}-trigger`, {
     ...vWaveTrigger,
     getSSRProps() {
       return {}
-    }
+    },
   })
 })

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   "module": "dist/index.js",
   "types": "dist/index.d.ts",
   "unpkg": "dist/index.global.js",
-  "files": ["dist/**/*", "nuxt/**/*"],
+  "files": ["dist/**/*", "nuxt/**/*", "src/options.ts"],
   "exports": {
     "./nuxt": {
       "import": "./nuxt/index.ts",

